bookkeeper-proto/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -90,8 +90,9 @@ message ReadRequest {
     enum Flag {
         FENCE_LEDGER = 1;
         ENTRY_PIGGYBACK = 2;
+        HIGH_PRIORITY = 3;
     }
-    optional Flag flag = 100;
+    repeated Flag flag = 100;
```

Comment:
> hmm I think this changes too much for a portback. i know repeated field is compatible with optional. However I am not sure how it would impact other existing code logic. I don't feel comfortable with change, especially this is coupled with a portback change.

The code changes required are forced by the change because the api for accessing repeated is completely different than that for accessing an optional.

I agree with what you're saying though, I was actually expecting this to fail BC in CI as enums in protobuf don't take kindly to new additions, so I was surprised when it past. My contingency was to add something like you suggest, though a boolean. An int is a good idea though. I'll leave everything else the same with regards to BookieClient.